app/assets/javascripts/angular/controllers/orders_controller.js

```diff
@@ -41,11 +41,17 @@ app.controller('DriversController', ['$scope', '$http', function($scope, $http)
   });
 
   dispatcher.bind('get_new_order', function(data) {
-    $http.get('/drivers/orders.json').success(function(data){
-      $scope.orders = data;
-    });
+    $scope.get_orders(1000);
   });
 
+  $scope.get_orders = function(delay) {
+    setTimeout(function() {
+      $http.get('/drivers/orders.json').success(function(data){
+        $scope.orders = data;
+      });
+    }, delay);
+  }
```

Comment:
When, for example, you assign driver for an order, he will got websocket message "You need download new informations from server course you have a new order". And some times this ws message + request to DB happening faster then the actual changes recieve DB and he download old information, not updated. To prevent this we can make delay, for expample, 1 second, and when he got message he will wait 1 second and then send request to server for new information.